samwise.js

```diff
@@ -5,34 +5,31 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-I'm impressed with your use of design patterns here, it really enhances the code organization.